src/MongoDB.Driver/Encryption/ClientEncryption.cs

```diff
@@ -39,11 +40,17 @@ public sealed class ClientEncryption : IDisposable
         /// <param name="clientEncryptionOptions">The client encryption options.</param>
         public ClientEncryption(ClientEncryptionOptions clientEncryptionOptions)
         {
-            _cryptClient = CryptClientCreator.CreateCryptClient(
+            var cryptClientSettings = new CryptClientSettings(
                 bypassQueryAnalysis: null,
+                csfleLibPath: null,
+                csfleSearchPath: null,
                 encryptedFieldsMap: null,
+                isCsfleRequired: false,
```

Comment:
I think it's better to make it null to highlight that it's not suitable here rather than disabled

Reply:
Done